RTN-004.tex

```diff
@@ -53,7 +57,320 @@
+\item To prepare the community to use the LSST data products and services, and especially to enable early science with commissioning and year 1 data.
+\end{enumerate}
+
+In order to achieve these goals, community participation in DP0 has two main components:
+(1) performing scientific exercises with the data products and services in the RSP, and
+(2) taking action to inform or enhance the performance of the DMS, the RSP, or the science community.
+
+Participants from the science community, "DP0 delegates", will be data rights holders\footnote{Data rights holders are US or Chilean scientists, or individuals associated with an International Contribution (in-kind) proposal in progress, as defined in \citeds{RDO-013}.} who represent a diversity of backgrounds.
+The term "delegates" reflects the fact that these individuals will be representing the science community as learners and as contributors to the achievement of the DP0 goals, as an essential step along the path towards the success of the LSST. 
+
+The nature of the two goal-driven components of DP0 participation listed above will vary according to individual delegates' interests and experience.
+
+\subsection{Two Stages: DP0.1 and DP0.2}\label{ssec:intro_stages}
+
+The simulated data set for DP0 was generated by the Dark Energy Science Collaboration (DESC) and is being used under an agreement between the DESC and Rubin.
+This data set is the second in a series of data challenges, and is thus named DC2.
+More information about DC2 can be found in the DESC DC2 paper \citep{2020arXiv201005926L} and on the DESC website\footnote{\url{https://lsstdesc.org/DC2-production/Documents/}}.
+
+The DP0.1 data set will contain the DC2 images and catalogs processed by the DESC.
+The DP0.1 data products will be available in the RSP in the format as processed by the DESC, which is similar to that of planned LSST data products.
+The DP0.1 data products that are made available to DP0 delegates will all be \emph{science-ready}, which means that they have passed full-scale validation and quality analysis.
+The projected date by which these DP0.1 science-ready data products will be made available to DP0 delegates in the RSP is June 30 2021.
+Therefore, in Section \ref{ssec:sel_time} the first stage of DP0 delegates are scheduled to have their accounts activated by June 30 2021.
+
+The DP0.2 data set will contain the DC2 images and catalogs reprocessed by the LSST Science Pipelines.
+The DP0.2 data products will be available in the RSP in a format that is consistent with the planned LSST data products.
+The projected date by which the DP0.2 science-ready data products will be made available to DP0 delegates in the RSP is March 31 2022. 
+Therefore, in Section \ref{ssec:sel_time} the second stage of DP0 delegates are scheduled to have their accounts activated by March 31 2022.
+
+Further details about the DP0.1 and DP0.2 data sets will be presented elsewhere.
```

Comment:
might want to specify where/when

Reply:
Good point thanks. In the tickets/preops-283 branch, I updated this to "Further details about the DP0.1 and DP0.2 data sets, such as their exact table schema and formats, will be presented as part of the resources and materials made available to DP0 delegates (Section \ref{ssec:res_sup}).".